src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp

```diff
@@ -1498,7 +1534,10 @@ class npc_valkyr_shadowguard : public CreatureScript
                 {
                     case POINT_DROP_PLAYER:
                         DoCastAOE(SPELL_EJECT_ALL_PASSENGERS);
-                        me->DespawnOrUnsummon(1000);
+                        if (IsHeroic())
+                            me->GetMotionMaster()->MoveTargetedHome();
```

Comment:
They now do move to the middle, to a random position with at a maximum of 4.0f from the center, this number may not be pretty accurate, but I don't think it can be deducted from sniffs